ui/Stars/HalfStar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const HalfStar: React.FC = () => {
```

Comment:
przecież to są 3 takie same komponenty z tylko innym kolorem :)
daj kolor jako props i zrób z tego 1 komponent